<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TikTag v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QStackedWidget" name="verticalStackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="verticalStackedWidgetPage1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="title">
            <string>General</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="albumArtLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelDuration">
                <property name="text">
                 <string>Duration: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelSampleRate">
                <property name="text">
                 <string>Sample Rate:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelChannels">
                <property name="text">
                 <string>Channels:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelBitrate">
                <property name="text">
                 <string>Bitrate:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCodec">
                <property name="text">
                 <string>Codec:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Metadata</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>35</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>35</number>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>25</number>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalStackedWidgetPage2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="bigAlbumArtLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>120</width>
             <height>120</height>
            </size>
           </property>
           <property name="flow">
            <enum>QListView::TopToBottom</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectFolder"/>
   <addaction name="actionDelete"/>
   <addaction name="actionCreateFolder"/>
   <addaction name="actionSetRootDir"/>
   <addaction name="actionLevelUp"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionTagToFile"/>
   <addaction name="actionFileToTag"/>
   <addaction name="actionDeleteTag"/>
   <addaction name="actionRevertFile"/>
   <addaction name="actionFolderByTag"/>
   <addaction name="actionGetOnlineTags"/>
   <addaction name="actionPutTagsOnline"/>
   <addaction name="actionSetSource"/>
   <addaction name="actionSpectogram"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelectFolder"/>
    <addaction name="actionCreateFolder"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionDelete"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTagToFile"/>
    <addaction name="actionFileToTag"/>
    <addaction name="actionFolderByTag"/>
    <addaction name="actionRevertFile"/>
    <addaction name="actionDeleteTag"/>
    <addaction name="actionSpectogram"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOnline">
    <property name="title">
     <string>Online</string>
    </property>
    <addaction name="actionGetOnlineTags"/>
    <addaction name="actionPutTagsOnline"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOnline"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSelectFolder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/open-folder.png</normaloff>:/icons/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>SelectFolder</string>
   </property>
  </action>
  <action name="actionTagToFile">
   <property name="text">
    <string>TagToFile</string>
   </property>
  </action>
  <action name="actionFileToTag">
   <property name="text">
    <string>FileToTag</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionDeleteTag">
   <property name="text">
    <string>DeleteTag</string>
   </property>
  </action>
  <action name="actionRevertFile">
   <property name="text">
    <string>RevertFile</string>
   </property>
  </action>
  <action name="actionFolderByTag">
   <property name="text">
    <string>FolderByTag</string>
   </property>
  </action>
  <action name="actionGetOnlineTags">
   <property name="text">
    <string>GetOnlineTags</string>
   </property>
  </action>
  <action name="actionCreateFolder">
   <property name="text">
    <string>CreateFolder</string>
   </property>
  </action>
  <action name="actionLevelUp">
   <property name="text">
    <string>LevelUp</string>
   </property>
  </action>
  <action name="actionPutTagsOnline">
   <property name="text">
    <string>PutTagsOnline</string>
   </property>
  </action>
  <action name="actionSpectogram">
   <property name="text">
    <string>Spectogram</string>
   </property>
  </action>
  <action name="actionSetRootDir">
   <property name="text">
    <string>SetRootDir</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSetSource">
   <property name="text">
    <string>SetSource</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
